From: Libor Pechacek <lpechacek@suse.cz>
Date: Fri, 12 Feb 2021 13:53:45 +0100
Subject: [PATCH] Fix compilation on 32bit platforms
Patch-upstream: no, submitted at https://github.com/intel/idxd-config/issues/3

While DSA and IAX are embedded in 64bit CPU, it's fair to assume
that the operating system might run in 32bit mode. This patch
makes the source code compile in 32bit environment.
---
 accfg/libaccel_config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/accfg/libaccel_config.h b/accfg/libaccel_config.h
index 0814601a9156..bed340d016ce 100644
--- a/accfg/libaccel_config.h
+++ b/accfg/libaccel_config.h
@@ -102,7 +102,7 @@ struct wq_parameters {
 	unsigned int priority;
 	int block_on_fault;
 	unsigned int max_batch_size;
-	unsigned long max_transfer_size;
+	uint64_t max_transfer_size;
 	const char *mode;
 	const char *type;
 	const char *name;
-- 
2.30.0

